x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -122,14 +121,78 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         expect(tableData.sort()).to.eql(expectedData.sort());
       });
 
+      describe('for the search bar', () => {
+        let adminSearchBar: WebElementWrapper;
+        let querySubmitButton: WebElementWrapper;
```

Comment:
nit: if these are not reused amongst test cases (i.e. not receive a value in a `beforeEach`), there is no need to declare them here

Reply:
since you re-define them in every case, they're practically local variables, so there is no real win in pushing them up to a larger scope. what would be a reason for having them there is to assign a value to them in a `beforeEach` hook, otherwise it's best to keep variable/const scope as small as possible in general